liquibase-couchbase/src/test/java/system/change/CreateCollectionSystemTest.java

```diff
@@ -19,6 +19,8 @@
 
 public class CreateCollectionSystemTest extends LiquibaseSystemTest {
 
+    private static final String COLLECTION_NAME = "travels";
+
     private final String travelsCollection = "travels";
```

Comment:
Do we need this one? What's the difference between `COLLECTION_NAME` and `travelsCollection` in terms of code logic?